build.gradle.kts

```diff
@@ -245,11 +245,7 @@ tasks {
 		filter {
 			includeTestsMatching("*Tests")
 		}
-		// java.security.manager was added in Java 12 (see
-		// https://www.oracle.com/java/technologies/javase/12-relnote-issues.html#JDK-8191053). We have to explicitly
-		// set it to "allow" for EnvironmentVariableUtilsTests$With_SecurityManager.
-		if (JavaVersion.current() >= JavaVersion.VERSION_12)
-			systemProperty("java.security.manager", "allow")
+		// Earlier builds enabled the Security Manager - it was removed in Java 24, so we deleted it.
```

Comment:
I wonder that this and the whole code gone below has no impact on documentation.